src/main/scala/is/hail/io/IndexBTree.scala

```diff
@@ -62,6 +68,12 @@ object IndexBTree {
   ): Unit = hConf.writeDataFile(fileName) { w =>
     w.write(btreeBytes(arr, branchingFactor))
   }
+
+  def toString(
```

Comment:
all the above changes and this was just for some debugging help and better test failure messages